<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MellowAVR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>811</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabInfo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>811</width>
      <height>191</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Information</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1121</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Compiler</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1121</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Serial Port</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>991</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>901</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>120</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New_Project"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_As"/>
    <addaction name="action_New"/>
    <addaction name="action_Print_Review"/>
    <addaction name="action_Print"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Copy"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Paste"/>
    <addaction name="action_Redo"/>
    <addaction name="action_Undo"/>
    <addaction name="action_Select_All"/>
   </widget>
   <widget class="QMenu" name="menu_Find">
    <property name="title">
     <string>&amp;Find</string>
    </property>
    <addaction name="action_Find_All"/>
   </widget>
   <widget class="QMenu" name="menu_Build">
    <property name="title">
     <string>&amp;Build</string>
    </property>
    <addaction name="action_Compile"/>
    <addaction name="action_Make"/>
    <addaction name="actionClean"/>
    <addaction name="actionDisassembler"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionGCC_version"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Find"/>
   <addaction name="menu_Build"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Paste"/>
   <addaction name="separator"/>
   <addaction name="action_Redo"/>
   <addaction name="action_Undo"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset>
     <normaloff>icons/add.png</normaloff>icons/add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset>
     <normaloff>icons/folder.png</normaloff>icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset>
     <normaloff>icons/disk.png</normaloff>icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="text">
    <string>&amp;Save As</string>
   </property>
  </action>
  <action name="action_Print_Review">
   <property name="text">
    <string>&amp;Print Review</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset>
     <normaloff>icons/copy.png</normaloff>icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset>
     <normaloff>icons/cut.png</normaloff>icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset>
     <normaloff>icons/edit-paste.png</normaloff>icons/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset>
     <normaloff>icons/redo.png</normaloff>icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset>
     <normaloff>icons/undo.png</normaloff>icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="action_Select_All">
   <property name="text">
    <string>&amp;Select All</string>
   </property>
  </action>
  <action name="action_Find_All">
   <property name="text">
    <string>&amp;Find All</string>
   </property>
  </action>
  <action name="action_Compile">
   <property name="text">
    <string>&amp;Compile</string>
   </property>
  </action>
  <action name="action_Make">
   <property name="text">
    <string>&amp;Make</string>
   </property>
  </action>
  <action name="action_New_Project">
   <property name="text">
    <string>&amp;New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionGCC_version">
   <property name="text">
    <string>&amp;avr-gcc version</string>
   </property>
   <property name="toolTip">
    <string>avr-gcc version</string>
   </property>
  </action>
  <action name="actionClean">
   <property name="text">
    <string>Clean</string>
   </property>
  </action>
  <action name="actionDisassembler">
   <property name="text">
    <string>Disassembler</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
